src/metabase/api/card.clj

```diff
@@ -601,16 +601,16 @@ saved later when it is ready."
 (defn- publish-card-update!
   "Publish an event appropriate for the update(s) done to this CARD (`:card-update`, or archiving/unarchiving
```

Comment:
:card-archive and :card-unarchive appear unused. Should we just remove them instead?

Reply:
I'd not change that here, I want to keep this PR only for adding tests